packages/communication-react/CHANGELOG.beta.md

```diff
@@ -7,34 +7,21 @@
-- Together mode client state and subscriber implementation ([PR #5152](https://github.com/azure/communication-ui-library/pull/5152) by nwankwojustin93@gmail.com)
-- Together mode client state and subscriber implementation ([PR #5152](https://github.com/azure/communication-ui-library/pull/5152) by nwankwojustin93@gmail.com)
-- Add Remote UFD new types to state ([PR #5348](https://github.com/azure/communication-ui-library/pull/5348) by jiangnanhello@live.com)
-- Together mode client state and subscriber implementation ([PR #5152](https://github.com/azure/communication-ui-library/pull/5152) by nwankwojustin93@gmail.com)
-- Together mode client state and subscriber implementation ([PR #5152](https://github.com/azure/communication-ui-library/pull/5152) by nwankwojustin93@gmail.com)
-- RTT component for single line of RTT ([PR #5356](https://github.com/azure/communication-ui-library/pull/5356) by 96077406+carocao-msft@users.noreply.github.com)
-- This PR contains implementation of together mode client state changes and the event listener for together mode stream updates ([PR #5152](https://github.com/azure/communication-ui-library/pull/5152) by nwankwojustin93@gmail.com)
-- RTT Disclosure Banner ([PR #5360](https://github.com/azure/communication-ui-library/pull/5360) by 96077406+carocao-msft@users.noreply.github.com)
-- Together mode client state and subscriber implementation ([PR #5152](https://github.com/azure/communication-ui-library/pull/5152) by nwankwojustin93@gmail.com)
-- Update API to allow for better identity control ([PR #5322](https://github.com/azure/communication-ui-library/pull/5322) by dmceachern@microsoft.com)
 ### Improvements
-- Updated Chat SDK and Signaling to Latest Stable Release ([PR #5361](https://github.com/azure/communication-ui-library/pull/5361) by 109105353+jpeng-ms@users.noreply.github.com)
-- On safari, do not show speaker dropdown ([PR #5328](https://github.com/azure/communication-ui-library/pull/5328) by 2684369+JamesBurnside@users.noreply.github.com)
+- Update Chat signaling to latest beta ([PR #5361](https://github.com/azure/communication-ui-library/pull/5361) by 109105353+jpeng-ms@users.noreply.github.com)
+- Stop showing speaker dropdown on Safari ([PR #5328](https://github.com/azure/communication-ui-library/pull/5328) by 2684369+JamesBurnside@users.noreply.github.com)
 - Upgrade calling stable version to 1.29.1 ([PR #5330](https://github.com/azure/communication-ui-library/pull/5330) by 79475487+mgamis-msft@users.noreply.github.com)
 ### Bug Fixes
-- add new strings to better announce the state of the participant ([PR #5345](https://github.com/azure/communication-ui-library/pull/5345) by 94866715+dmceachernmsft@users.noreply.github.com)
+- Add new strings to better announce the state of the participant ([PR #5345](https://github.com/azure/communication-ui-library/pull/5345) by 94866715+dmceachernmsft@users.noreply.github.com)
```

Comment:
"Add new Aria strings to better support narration of participant information in the video gallery" (seems like one of my changelogs or something...)

Reply:
Updated! Thanks.